metadata-ingestion/src/datahub/cli/docker_cli.py

```diff
@@ -610,7 +604,6 @@ def detect_quickstart_arch(arch: Optional[str]) -> Architectures:
 @telemetry.with_telemetry
 def quickstart(
     version: str,
-    build_locally: bool,
```

Comment:
Are we deprecating this fully? I don't know how much usage it gets

Reply:
People shouldn't use the build-locally option here, they should instead use ./gradlew quickstart